os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -440,14 +441,12 @@ static void arm_assert(void)
 		up_mdelay(250);
 #endif
 		}
+#ifndef CONFIG_BOARD_ASSERT_SYSTEM_HALT
+	} else {
```

Comment:
This needs to be like below:
```
       if(xxx) {
       }
#ifndef CONFIG_BOARD_ASSERT_SYSTEM_HALT
	else {
		exit(errorcode);
	}
#endif
```

Reply:
Thank you for review~

Please look below codes,
The start conditional statement is also disabled by config.
So, It don't need changes.

```
#ifndef CONFIG_BOARD_ASSERT_SYSTEM_HALT
	/* Are we in an interrupt handler or the idle task? */

	if (CURRENT_REGS || (this_task())->flink == NULL) {
#endif
```

```
#ifndef CONFIG_BOARD_ASSERT_SYSTEM_HALT
	} else {
		exit(errorcode);
	}
#endif
```

Reply:
@kishore-sn If it's ok, could you leave an approval?

Reply:
@kishore-sn Could you check this?